docs/SQL/execution/execution.md

```diff
@@ -14,6 +14,21 @@ tags:
 
 You can leverage Data Copilot for SQL by building your model and adding individual gems, and then check how the process is going by interactively running the model. Use the Data Explorer to dig into your interim data sample, with the ability to sort, filter, and download your sample data.
 
+## Running a model
+
+There are two ways to run a model interactively:
+
+- Click the **play** button on the model canvas to run the entire model.
+- Click the **play** button on a particular gem to execute the flow **up to and including that gem**. This is useful for testing and debugging when you don't want to run the entire model.
+
+![Interactive run options](img/interactive-execution-sql.png)
+
+:::note
+
+When you add a new seed to a model, run the model using the play button on the model canvas to save that seed.
```

Comment:
Both of the play buttons are on the model canvas...to make this super clear, i suggest: 

"When you add a new seed to a model, run the model using the (1) BIG play button on the model canvas to save that seed. Then run using the (2) small play button to see a data sample.

Reply:
Edited: 
"When you create a new seed, paste the seed contents into the code view. Save the seed by clicking the large play button _inside the seed._ Open the model of interest and execute (run) the Seed using the (2) small play button.